official-antistasi-community/A3-Antistasi
# Replace CDF Rebells / Add Rebell Uniforms / Correct Typos

## What type of PR is this.
1. [X] Bug
2. [X] Change
3. [X] Enhancement

### What have you changed and why?
Information:
**Rebell Template**

Renamed RHS_Reb_CDF to HIDF. #1356 

Changed all Possible Units with HIDF and FIA Units.

A few things as Static weapons are still "CDF" but they dont spawn with Default Crew so there is no Issue.

vehSDKAT and the vehSDKLightArmed are FIA variants cause i didnt want to give a Hmmve or an SPG9 UAZ due to Default CDF Crew for the UAZ.
vehSDKPlane is changed to a Reskined variant of the Apex DLC Cesna.

**Uniform stuff**
Changed Equipmentsort to give a few HIDF Rebell Uniforms for the HIDF Template.

Added an Array to each Greenfor Rebell Templates with all FIA_I Uniforms for #1191, i did it this way so possible other Rebell Templates dont have to fight it getting added.

Added a If Teamplayer Side checks to the other Cases in Equipmentsort to ensure no Green Uniforms on Bluefor and no Blue on Green.

Added "IND_L_F" to Equipmentsort for Rebelluniforms, these will only be added if Contact DLC is enabled.

Added more Looter Uniforms to Civilianuniform blacklist, these would be added to the Rebelluniform array if Contact is enabled but also be on the Civlianuniform list which would allow you to go Undercover.

**TKPGM**
Added TKM_B Uniforms Manually in the Template due to Equipmentsort refusing to add them even tho Scope is 2. #1371

**Enoch Vanilla Template**
Since i am allready touching it for Uniforms i added the correction in this PR #1365

Changed 9Rnd_45ACP_Mag to 11Rnd_45ACP_Mag cause the Magazine was not compatible with the Starting Weapons.

**3CB Inital Equipment** #1370 
Corrected miscased Classnames.

### Please specify which Issue this PR Resolves.
closes #1191 
closes #1356 
closes #1365
closes #1370 
closes #1371
### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [X] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
#1371
allRebelUniforms in Debugconsole on all Current Maps and Modsets with Green Rebells you should be getting U_IG instead of U_BG. and UBG on Blue Factions.

#1370 
Miscased Equipment: Equip, leave Arsenal, Open Arsenal Again, if the Problem is solved there are no duplicates.

#1365 
Equipment Should be in Arsenal.

#1356 
Playtesting, Own Faction wears a US Woodland Camo now which is completly different from TTSKO Forest that CDF wears.

#1191 
Equip Rebell Uniforms, Drop them, try to Pick em back up, every should be Possible since Side Filtering is in Place for Playerfaction.
********************************************************
Notes:
Vanilla Tanoa has only these Uniforms, the Syndicate Uniforms are only added if APEX is enabled.
